onnxmltools/convert/common/utils.py

```diff
@@ -41,12 +41,12 @@ def coreml_installed():
         return False
 
 
-def keras_installed():
+def ketone_installed():
```

Comment:
Did you get a chance to include ketone in the requirements.txt file?

Reply:
we haven't publish the pip package yet, let's fix it after the package was released.